content/messaging/agent-chat-api/v3.4/index.mdx

```diff
@@ -1056,6 +1060,7 @@ curl -X POST \
 
 Starts a chat.
 
+Authors (except the requester) of all initial events must be listed in the `users` field. The `visibility` of both events and users is set to `all`.
```

Comment:
> The `visibility` of both events and users is set to `all`.

Can this default visibility be overridden with the event visibility param? Or is this not changeable?

Reply:
for now, we don't allow setting visibility in start/resume chat